site/docs/tasks/multicluster.md

```diff
@@ -106,3 +106,80 @@ For the remote cluster the coordinator will be disabled:
 coordinator:
     enabled: false
 ```
+
+
+## Configure Vault for multi-cluster deployment
+
+The Fybrik uses [HashiCorp Vault](https://www.vaultproject.io/) to provide running Fybrik modules in the clusters with the dataset credentials when accessing data. This is done using [Vault plugin system](https://www.vaultproject.io/docs/internals/plugins) as described in [here](../concepts/vault_plugins.md).
+
+
+This section describe the steps to enable the modules to authenticate to Vault in order for them to retrieve the dataset credentials.
+Some of the steps described below are not specific to the Fybrik project but rather are Vault specific and can be found in Vault related online tutorials.
```

Comment:
can be found in Vault's online tutorials.

Reply:
To be really picky, "Vault-related online tutorials" is better than "Vault's online tutorials" since this also includes all those online tutorials not produced by Vault (Hashicorp)